benchmarks/graphics_pipeline/main.cpp

```diff
@@ -30,6 +31,50 @@
 
 using namespace ppx;
 
+class Person
```

Comment:
Maybe this could be a `ControllableCamera` or `FlyCamera` or `FreeCamera`?
And inherit from the camera class, to which you add the move function?

And seems like rate of move is stored, but not uses, instead you do `Person.Move(direction, Person.GetRateOfMove())`.
Could this rate of move be a constant?

So you'd have `mCamera.Move(direction, kCameraSpeed * deltatime)` ?

Reply:
Done.